config/config.properties

```diff
@@ -223,6 +223,15 @@ bookkeeper.zkAddress=master.mesos:2181
 # This value must be the same for all Pravega SegmentStore instances in this cluster.
 #bookkeeper.zkMetadataPath=/segmentstore/containers
 
+# The maximum number of concurrent writes to BookKeeper. Writes are queued and will be executed in order.
+# Recommended values: 1-10. The best value will depend a lot on the installation details and BookKeeper capacity and
+# should be chosen empirically, based on performance tests.
+#bookkeeper.maxConcurrentWrites=5
```

Comment:
We should try to eventually get rid of this parameter, it sounds like a magic number (hard to figure out the exact value and justify).